components/contribution-flow/ContributionBlocker.js

```diff
@@ -83,15 +94,18 @@ export const getContributionBlocker = (loggedInUser, account, tier, shouldHaveTi
     return { reason: CONTRIBUTION_BLOCKER.TIER_MISSING, type: 'warning', showOtherWaysToContribute: true };
   } else if (tier && isTierExpired(tier)) {
     return { reason: CONTRIBUTION_BLOCKER.TIER_EXPIRED, type: 'warning', showOtherWaysToContribute: true };
-  } else if (account.settings.disableCustomContributions && !tier) {
```

Comment:
I don't get how this works, since `ContributionBlocker` will not be rendered if `getContributionBlocker` returns `null`